src/gameStatus.js

```diff
@@ -0,0 +1,7 @@
+export const CONTINUE_PLAYING = "Keep playing...";
+export const DRAW = "It's a draw!";
+
+export function winner(move) {
+  return 'Player ' + move + ' wins!';
```

Comment:
I'd name this starting with a verb again, als I'd probably use string interpolation here `\`Player ${move} wins!\``